runtime/src/mm/cpp/ObjectFactory.cpp

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
- * that can be found in the LICENSE file.
- */
-
-#include "ObjectFactory.hpp"
-
-#include "Alignment.hpp"
-#include "Alloc.h"
-#include "GlobalData.hpp"
-#include "Types.h"
-
-using namespace kotlin;
-
-ObjHeader* mm::ObjectFactory::ThreadQueue::CreateObject(const TypeInfo* typeInfo) noexcept {
```

Comment:
Why do we need these methods in a header instead of a cpp? Due to performance reasons?

Reply:
The entire `ObjectFactory` class has become a template (with a `typename GC` parameter), so all the definitions need to become visible.

Reply:
I see, thanks!